AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -159,6 +165,12 @@ - (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionVi
   return self;
 }
 
+-(void)dealloc {
+  // a little defense move here.
```

Comment:
Maybe reference the github issue # here, or add a little more substance :).  Great comments overall.